### ISSUE TITLE
Can't setup 9.975% tax rate - Québec Sales Tax (QST)

### ISSUE THREAD

**Issue by alexdx-ca** [medusajs/medusa#5048]
### Describe the bug

I can't setup Québec Sales Tax (QST). Because it's value is 9.975%
But Medusa is asking me 9.97% or 9.98%.

### System information

Medusa version: 1.3.20
Node.js version: 
Database: PostgreSQL 15.4
Operating system: macOS 12.6.1
Browser (if relevant): Firefox 

### Steps to reproduce the behavior

1. Go to http://localhost:7001/a/settings
2. Click on Taxes.
3. Click on "+ New Tax Rate" button.
4. Enter name, tax rate (9.975) and tax code.
5. Click "Create" and see error.

### Expected behavior

Tax rate field have to accept more than 2 decimal places.

### Screenshot

<img width="757" alt="Screen Shot 2023-09-13 at 6 04 12 PM" src="https://github.com/medusajs/medusa/assets/7256258/e1cc5190-4b7b-4ccd-a8a5-06d11b1b29ef">

**Comment by pevey** [comment]
Just adding info that this is not a limitation on the database.  I can manually add a tax rate with high specificity.  It might be just a matter of the admin app validators.

Have you tried adding the rate via the medusa API?  If it works, that or adding to the database directly could be a workaround until this is fixed.

**Comment by SalahAdDin** [comment]
@alexdx-ca  any update on this?

**Comment by alexdx-ca** [comment]
Just tried latest version of Medusa!
The issue is still there, form accepts only 2 decimal places.
So if possible, can you fix it please for future versions.
Thank you so much for your time and help!

**Comment by SalahAdDin** [comment]
Need to ping someone from the staff then,

**Comment by SalahAdDin** [comment]
@alexdx-ca anything here?

**Comment by SalahAdDin** [comment]
@alexdx-ca anything here?

**Comment by AmbroziuBaban** [comment]
@SalahAdDin . I've created an PR which will allow percentage value with 4 decimals. Now I'm waiting the code review.

**Comment by SalahAdDin** [comment]
> [@SalahAdDin](https://github.com/SalahAdDin) . I've created an PR which will allow percentage value with 4 decimals. Now I'm waiting the code review.

@AmbroziuBaban was it fixed?

**Comment by SalahAdDin** [comment]
> [@SalahAdDin](https://github.com/SalahAdDin) . I've created an PR which will allow percentage value with 4 decimals. Now I'm waiting the code review.

@AmbroziuBaban was it fixed?

**Comment by AmbroziuBaban** [comment]
> > [@SalahAdDin](https://github.com/SalahAdDin) . I've created an PR which will allow percentage value with 4 decimals. Now I'm waiting the code review.
> 
> [@AmbroziuBaban](https://github.com/AmbroziuBaban) was it fixed?

The PR was not merged. I'll open another one and contact the team to get it merged.

**Comment by AmbroziuBaban** [comment]
The initial PR was closed because it had no activity. I opened a new one.

**Comment by SalahAdDin** [comment]
> The initial PR was closed because it had no activity. I opened a new one.

Close this issue then, plz!

**Comment by AmbroziuBaban** [comment]
The issue is fixed by this [PR](https://github.com/medusajs/medusa/pull/13407) and can be closed.

**Comment by willbouch** [comment]
Yes! I indeed fixed it with the linked PR. Thank you and will be closing the issue

**Comment by SalahAdDin** [comment]
> The issue is fixed by this [PR](https://github.com/medusajs/medusa/pull/13407) and can be closed.
Finally, a bug was fixed!

**Comment by olivermrbl** [comment]
@SalahAdDin, if you have nothing nice to say, please keep it to yourself. We are triaging issues as fast as we can, and comments like these do nothing but discourage us.
